cmd/tsconnect/wasm/wasm_js.go

```diff
@@ -256,13 +255,42 @@ func (i *jsIPN) logout() {
 	go i.lb.Logout()
 }
 
-func (i *jsIPN) ssh(host, username string, termConfig js.Value) {
-	writeFn := termConfig.Get("writeFn")
-	setReadFn := termConfig.Get("setReadFn")
-	rows := termConfig.Get("rows").Int()
-	cols := termConfig.Get("cols").Int()
-	onDone := termConfig.Get("onDone")
+func (i *jsIPN) ssh(host, username string, termConfig js.Value) map[string]any {
+	jsSSHSession := jsSSHSession{
```

Comment:
add a `&` here. Your Run method's on the pointer receiver here, so don't make it possible to accidentally do value copies in the future. Stay in pointers from the beginning.